src/SFA.DAS.Reservations.Web/Views/ProviderReservations/ChooseEmployer.cshtml

```diff
@@ -39,16 +42,7 @@
                     <td class="govuk-table__cell">@employer.AccountName</td>
                     <td class="govuk-table__cell">@employer.AccountLegalEntityPublicHashedId</td>
                     <td class="govuk-table__cell govuk-table__cell--numeric">
-                        <a class="govuk-link" href="@Url.Action("ConfirmEmployer", new
-                                 {
-                                     UkPrn = ViewContext.RouteData.Values["ukPrn"],
-                                     AccountId = employer.AccountId,
-                                     AccountPublicHashedId = employer.AccountPublicHashedId,
-                                     AccountName = employer.AccountName,
-                                     AccountLegalEntityId = employer.AccountLegalEntityId,
-                                     AccountLegalEntityPublicHashedId = employer.AccountLegalEntityPublicHashedId,
-                                     AccountLegalEntityName = employer.AccountLegalEntityName
-                                 })">Select <span class="govuk-visually-hidden">@employer.AccountLegalEntityName</span></a>
+                        <a class="govuk-link" href="@Url.Action("ConfirmEmployer", viewModel)">Select <span class="govuk-visually-hidden">@employer.AccountLegalEntityName</span></a>
```

Comment:
have we got the route name defined this we can use instead? `asp-route` ?

Reply:
yep, will do.